conductor-client/src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -598,42 +604,45 @@ class ExternalDatabaseManagementService(
         return permissions
     }
 
-    private fun updateHBARecords(dbName: String) {
+    private fun updateHBARecords(organizationId: UUID) {
         val originalHBAPath = Paths.get(organizationExternalDatabaseConfiguration.path + organizationExternalDatabaseConfiguration.fileName)
         val tempHBAPath = Paths.get(organizationExternalDatabaseConfiguration.path + "/temp_hba.conf")
 
         //create hba file with new records
-        val records = getHBARecords(PostgresTable.HBA_AUTHENTICATION_RECORDS.name)
-                .map {
-                    it.buildHBAConfRecord()
-                }.toSet()
-        try {
-            val out = BufferedOutputStream(
-                    Files.newOutputStream(tempHBAPath, StandardOpenOption.WRITE, StandardOpenOption.CREATE,
-                            StandardOpenOption.TRUNCATE_EXISTING)
-            )
-            records.forEach {
-                val recordAsByteArray = it.toByteArray()
-                out.write(recordAsByteArray)
-            }
-            out.close()
+        val records = getHBARecords(PostgresTable.HBA_AUTHENTICATION_RECORDS.name).map {
+            it.buildHBAConfRecord()
+        }.toSet()
 
-            //reload config
-            externalDbManager.connect(dbName).connection.use { conn ->
-                val stmt = conn.createStatement()
-                stmt.executeQuery(getReloadConfigSql())
+        try {
+            Files.newOutputStream(
+                    tempHBAPath,
+                    StandardOpenOption.WRITE,
+                    StandardOpenOption.CREATE,
+                    StandardOpenOption.TRUNCATE_EXISTING
+            ).use { `in` ->
+                BufferedOutputStream(`in`).use { out ->
+                    records.forEach {
+                        val recordAsByteArray = it.toByteArray()
+                        out.write(recordAsByteArray)
+                    }
+                }
             }
         } catch (ex: IOException) {
             logger.info("IO exception while creating new hba config")
         }
 
+        //reload config
+        externalDbManager.connectToOrg(organizationId).connection.use { conn ->
+            val stmt = conn.createStatement()
```

Comment:
should probably `stmt.user { stmt -> ... }`

Reply:
derp, yeah I did this once and then merged badly